mscore/importmxmlpass1.cpp

```diff
@@ -542,7 +542,7 @@ static void doCredits(Score* score, const CreditWordsList& credits, const int pa
              pf->width(), pf->height(), pf->oddTopMargin(), score->spatium(), DPMM, DPI);
       */
       // page width, height and odd top margin in tenths
-      const double ph  = score->styleD(Sid::pageHeight) * 10 * DPI / score->spatium();
+      const double ph = score->style().pageOdd().heightPoints() * 10 / score->spatium();
```

Comment:
My constant `TEN`, defined in exportxml.cpp, should be used here.  I will need to include exportxml.cpp here.  I don't think that's a problem.  With that change, I might be able to use one of the `getTenths` functions here instead.  I will work on it...

Reply:
I replace TEN with SPATIUM_XML and moved it to style.h